src/System.IO/tests/BinaryWriter/BinaryWriter.WriteByteCharTests.cs

```diff
@@ -558,5 +558,40 @@ public void BinaryWriter_WriteCharArrayTest3()
             mstr.Dispose();
             dw2.Dispose();
         }
+
+        [Fact]
+        public void BinaryWriter_WriteSpan()
```

Comment:
You need to put the test in a file that's only included in the netcoreapp build, e.g. conditioned like this file:
https://github.com/jp2masa/corefx/blob/6546a430e9d05f52157c0a426fbb1c70d3662835/src/System.IO/tests/System.IO.Tests.csproj#L43